lib/train/platforms/uuid.rb

```diff
@@ -0,0 +1,135 @@
+# encoding: utf-8
+
+require 'digest/sha1'
+require 'securerandom'
+require 'json'
+
+module Train::Platforms
+  class UUID
+    def initialize(platform)
+      @platform = platform
+      @backend = @platform.backend
+    end
+
+    def find_or_create_uuid
+      if @platform.unix?
+        unix
+      elsif @platform.windows?
+        windows
+      elsif @platform.name == 'aws'
+        aws
+      elsif @platform.name == 'azure'
```

Comment:
Why aren't there predicate methods for aws and azure already?

Reply:
due to <name>? only getting created for platform families. I have been thinking about this and I will wrap aws/azure in their own families so we get them.

Reply:
Gotcha. Yeah it'd be great to have the uniform api. Then the `case` statement could be collapsed. =^)